l3kernel/l3str.dtx

```diff
@@ -843,6 +843,16 @@
 %   code and so should only be used for short-term storage.
 % \end{variable}
 %
+% \section{Deprecated functions}
+%
+% \begin{function}[EXP, added = 2019-11-26]
+%   {\str_foldcase:n, \str_foldcase:V}
+%   \begin{syntax}
+%     \cs{str_foldcase:n} \Arg{tokens}
+%   \end{syntax}
+%   A previous name for the functionally-identical \cs{str_casefold:n}.
+% \end{function}
+%
```

Comment:
Weren't we completely dropping documentation for deprecated functions?

Reply:
That was the case when we were removing deprecated functions over time. We've now said we won't, so I've been leaving the docs around as they will never disappear. I'm happy to remove if that feels like a better plan.

Reply:
Deprecated functions are for code that still uses them, not for newer code, so they don't really need documentation (as a way to discourage their usage).  People will keep using deprecated functions in new code if they are documented.

What could be useful instead would be a “migration manual” so that people looking for the documentation of deprecated functions will find a way to replace them with the correct versions

Reply:
Perfectly reasonable point: do we want that in this PR or separately?

Reply:
I think a separate PR to keep it on topic.  I can draft that.

Does a separate section at the end of interface3 sound good?

Reply:
It can go in after candidates I guess: we have 'Additions and Removals' for that part anyway.